source/aggregation-pipeline-builder.txt

```diff
@@ -73,8 +73,62 @@ Limitations
 Create an Aggregation Pipeline
 ------------------------------
 
+Aggregation pipelines can be created in 2 seperate modes:
+
+- Stage View Mode, a visual click based pipeline editor which 
+  preloads pipeline syntax based on user selections.
```

Comment:
Maybe instead of "user", we can use second person to talk directly to the reader. As in:

"preloads pipeline syntax based on your selected stages."

or something like that.